db/migrate/20200930085956_create_osbbs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOsbbs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :osbbs do |t|
```

Comment:
Where are indexes, constraints...?

Reply:
I have index for name and validations before save object osbb in database.

Reply:
I see